scripts/bootstrap/compile.sh

```diff
@@ -281,6 +281,10 @@ EOF
   link_file "${PWD}/tools/cpp/runfiles/BUILD.tools" \
       "${BAZEL_TOOLS_REPO}/tools/cpp/runfiles/BUILD"
 
+  # Create @bazel_tools//tools/cpp/cpp20modules_tools
+  mkdir -p ${BAZEL_TOOLS_REPO}/tools/cpp/cpp20modules_tools
+  link_file "${PWD}/tools/cpp/cpp20modules_tools/BUILD.tools" "${BAZEL_TOOLS_REPO}/tools/cpp/cpp20modules_tools/BUILD"
```

Comment:
nit: use `//tools/cpp/cpp_modules` directory (to follow the naming change done in [1/5])